<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1705</width>
    <height>1065</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Frame_Inferior_Izq">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>200</width>
      <height>811</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color:#a8dadc;
color:black;
border-style:outset;
border-width:2px;
border-radius:0px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}


</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:#a8dadc;}

QPushButton{
background-color:#a8dadc;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
font: 75 12pt &quot;Arial Narrow&quot;;
}
QPushButton:hover{
background-color:#f1faee;
color:black;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}</string>
       </property>
       <property name="text">
        <string>Historia</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes_Interfaz/history-solid.svg</normaloff>Imagenes_Interfaz/history-solid.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_12">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:#a8dadc;}

QPushButton{
background-color:#a8dadc;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
font: 75 12pt &quot;Arial Narrow&quot;;
}
QPushButton:hover{
background-color:#f1faee;
color:black;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}
</string>
       </property>
       <property name="text">
        <string>Bases de datos</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes_Interfaz/magnifying-glass-solid.svg</normaloff>Imagenes_Interfaz/magnifying-glass-solid.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:#a8dadc;}

QPushButton{
background-color:#a8dadc;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
font: 75 12pt &quot;Arial Narrow&quot;;
}
QPushButton:hover{
background-color:#f1faee;
color:black;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}</string>
       </property>
       <property name="text">
        <string>Conceptos</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes_Interfaz/chalkboard-teacher-solid.svg</normaloff>Imagenes_Interfaz/chalkboard-teacher-solid.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:#a8dadc;}

QPushButton{
background-color:#a8dadc;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
font: 75 12pt &quot;Arial Narrow&quot;;
}
QPushButton:hover{
background-color:#f1faee;
color:black;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}
</string>
       </property>
       <property name="text">
        <string>CRUD</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes_Interfaz/database-solid.svg</normaloff>Imagenes_Interfaz/database-solid.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:#a8dadc;}

QPushButton{
background-color:#a8dadc;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
font: 75 12pt &quot;Arial Narrow&quot;;
}
QPushButton:hover{
background-color:#f1faee;
color:black;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}</string>
       </property>
       <property name="text">
        <string>Datasets</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes_Interfaz/table-solid.svg</normaloff>Imagenes_Interfaz/table-solid.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_10">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:#a8dadc;}

QPushButton{
background-color:#a8dadc;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
font: 75 12pt &quot;Arial Narrow&quot;;
}
QPushButton:hover{
background-color:#f1faee;
color:black;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}</string>
       </property>
       <property name="text">
        <string>Pokemon Test</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes_Interfaz/Poké_Ball_icon.svg</normaloff>Imagenes_Interfaz/Poké_Ball_icon.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:#a8dadc;}

QPushButton{
background-color:#a8dadc;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
font: 75 12pt &quot;Arial Narrow&quot;;
}
QPushButton:hover{
background-color:#f1faee;
color:black;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}</string>
       </property>
       <property name="text">
        <string>Netflix Test</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes_Interfaz/netflix-seeklogo.com.svg</normaloff>Imagenes_Interfaz/netflix-seeklogo.com.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_30">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:#a8dadc;}

QPushButton{
background-color:#a8dadc;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
font: 75 12pt &quot;Arial Narrow&quot;;
}
QPushButton:hover{
background-color:#f1faee;
color:black;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}</string>
       </property>
       <property name="text">
        <string>Chile Test</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes_Interfaz/Flag_of_Chile.svg</normaloff>Imagenes_Interfaz/Flag_of_Chile.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:#a8dadc;}

QPushButton{
background-color:#a8dadc;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
font: 75 12pt &quot;Arial Narrow&quot;;
}
QPushButton:hover{
background-color:#f1faee;
color:black;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}</string>
       </property>
       <property name="text">
        <string>Calculadora T(n)</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes_Interfaz/calculator-solid.svg</normaloff>Imagenes_Interfaz/calculator-solid.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
background-color:#a8dadc;}

QPushButton{
background-color:#a8dadc;
border-top-left-radius: 20px;
border-bottom-left-radius: 20px;
font: 75 12pt &quot;Arial Narrow&quot;;
}
QPushButton:hover{
background-color:#f1faee;
color:black;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}</string>
       </property>
       <property name="text">
        <string>Conclusiones</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes_Interfaz/bullseye-solid.svg</normaloff>Imagenes_Interfaz/bullseye-solid.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>194</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">border-color: #a8dadc;</string>
       </property>
       <property name="text">
        <string>Mathias Cáceres B.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>1251</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Imagenes/db.jpg);
QFrame{
background-color:#457b9d;
font:87 12pt&quot;Arial Black&quot;;
color:black;
border-style:outset;
border-width:2px;
border-radius:0px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>850</y>
      <width>1611</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color:rgb(69, 123, 157);
font:87 12pt&quot;Arial Black&quot;;
color:black;
border-style:outset;
border-width:2px;
border-radius:0px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>0</y>
      <width>251</width>
      <height>881</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color:rgb(69, 123, 157);
font:87 12pt&quot;Arial Black&quot;;
color:black;
border-style:outset;
border-width:2px;
border-radius:0px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="Frame_Superior">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1511</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color:rgb(69, 123, 157);
font:87 12pt&quot;Arial Black&quot;;
color:black;
border-style:outset;
border-width:2px;
border-radius:0px;
border-color:f1faee;
font:bold 14 px;
padding:10px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_menu">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:#457b9d;
font:87 12pt&quot;Arial Black&quot;;
border-radius:2px;
}
</string>
       </property>
       <property name="text">
        <string>MENÚ</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Imagenes/bars-solid.svg</normaloff>../../../Imagenes/bars-solid.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>Frame_Inferior_Izq</zorder>
   <zorder>label_2</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>Frame_Superior</zorder>
  </widget>
 </widget>
 <resources>
  <include location="Historia Edicion.qrc"/>
 </resources>
 <connections/>
</ui>
